src/pages/Translator.js

```diff
@@ -133,7 +117,14 @@ export default function Translator() {
             style: { width: '100%', justifyContent: 'center' }
           }, [Spinner]),
 
-          div({ isRendered: !isLoading, className: 'row no-margin' }, [
+          div({ isRendered: error }, [
+            'There was an error running your request'
```

Comment:
do we want to use error to show something more specific

Reply:
i dont want to show the error message if it's something wrong with our servers or our code, only if it's a user error (that they can do something about). For now, I think this is okay, since I'm just trying to rush this code in real quick ^_^;